### ISSUE TITLE
Spinner should have aria-label from progress-bar role

### ISSUE THREAD

**Issue by brianseek** [CMSgov/design-system#974]
**Describe the bug**
Ensure that each element with role="progressbar" has one of the following characteristics:

- Non-empty aria-label attribute.
- aria-labelledby pointing to element with text which is discernible to screen reader users.

Read more here: https://dequeuniversity.com/rules/axe/4.1/aria-progressbar-name

**To Reproduce**
View the markup for the Spinner component

**Comment by davidakennedy** [comment]
Good catch! Thank you, @brianseek.

Some more context:

- This PR has a bit of history around this component and accessibility features: https://github.com/CMSgov/design-system/pull/138
- You can reproduce this issue with the Axe browser extension.
- In my quick testing, all major desktop screen readers announce the current `aria-valuetext="Loading"`, but that doesn't get mapped to the accessible name in the accessibility tree.
- In more quick testing, I added an `aria-label` value, which appears to only be announced (in addition to the `aria-valuetext`) in Safari/VoiceOver.
- I'd suggest exploring adding an `aria-label` in addition to the existing `aria-valuetext. That would satisfy the accessibility tree requirement, and it looks like not be overly verbose to screen readers.

**Comment by davidakennedy** [comment]
Filed in Jira: WNMGDS-843.

**Comment by pwolfert** [comment]
This was completed under https://github.com/CMSgov/design-system/pull/1131. Linked to that pull request and closing.
